fastlane/Fastfile

```diff
@@ -238,6 +238,13 @@ platform :ios do
 
   desc "visionOS build"
   lane :build_visionos do |options|
+    pod_lib_lint(
+      verbose: true,
+      podspec:'RevenueCat.podspec', 
+      platforms:"xros",
+      fail_fast: true
+    )
+
```

Comment:
feels like we should also include it here: https://github.com/RevenueCat/purchases-ios/blob/main/fastlane/Fastfile#L937 and here https://github.com/RevenueCat/purchases-ios/blob/main/fastlane/Fastfile#L226

Reply:
unless xcode versions pose an issue

Reply:
Yeah I made it separate until we use Xcode 15 for everything. I'll leave a comment.

Reply:
Comment added.